src/lsp_data.rs

```diff
@@ -262,6 +262,22 @@ impl Default for InitializationOptions {
     }
 }
 
+// Subset of flags from ls_types::ClientCapabilities that affects this RLS.
+// Passed in the `initialize` request under `capabilities`.
+#[derive(Debug, PartialEq, Deserialize, Serialize, Clone, Copy)]
+#[serde(default)]
+pub struct HandledClientCapabilities {
+    pub has_snippet_support: bool,
```

Comment:
This is specific to code completion, right? Could the name reflect that?

Reply:
ok